jmclient/jmclient/wallet-rpc-api.yaml

```diff
@@ -0,0 +1,649 @@
+openapi: 3.0.0
+info:
+  description: Joinmarket wallet API
+  version: "1"
+  title: Joinmarket wallet API
+paths:
+  /wallet/create:
+    post:
+      summary: create a new wallet
+      operationId: createwallet
+      description: Give a filename (.jmdat must be included) and a password, create the wallet and get back the seedphrase for the newly persisted wallet file. The wallettype variable must be one of "sw" - segwit native, "sw-legacy" - segwit legacy or "sw-fb" - segwit native with fidelity bonds supported, the last of which is the default. 
+      responses:
+        '201':
+          $ref: '#/components/responses/Create-201-OK'
+        '401':
```

Comment:
I'm not sure what I had in my head, but I decided that I didn't need to add the `BadRequest` to all the response lists; I think I thought that way, because surely it's possible for the request to be badly formed for any route?

But now I say it, I immediately realise it's not true: if a route has no parameters at all attached, it's impossible for there to be a bad request I think. So I'll add it in the spec, to all the ones that have parameters and make sure we're always catching with `InvalidRequest` in those cases. Thanks.

Reply:
> This also returns 401 Wallet Already Unlocked

I don't believe `createwallet` does, though?

Reply:
I can reproduce it, isn't it this code [here](https://github.com/abhishek0405/joinmarket-clientserver/blob/0e37ed52c70efa6fbc6a969e626df0fde70f70fd/jmclient/jmclient/wallet_rpc.py#L504-L511)?

Reply:
Yes, sorry, got confused there.

Reply:
As per above, added 400 responses to all the routes that return it in commit https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/996/commits/70e33afbd4d6fef8c579bd64e7b8c4bcc585e261